docs/account_system.md

```diff
@@ -0,0 +1,158 @@
+Account system
+==============
+
+The account management is homebrew, except the password hashing which is handled by [bcrypt](https://godoc.org/golang.org/x/crypto/bcrypt) (adaptative hashing algorithm).
+
+Expected client behavior
+------------------------
+
+The user can choose to register an account or login to an existing one. Both return a session key (see API below), which must be stored locally (via cookies for instance).
+
+Session keys expire after a certain time spent without being used, or if 5 other sessions have been opened since the last time the key has been used. If a key is expired, a re-login is needed.
```

Comment:
Probably OAuth-style tokens are the way to go -- for instance, a third party service may want to store a token that provides limited account access (CloudPebble, for instance, needs access to 'verify that this phone goes with this login', but should not be able to change a user's e-mail address or password).